File: scripts/dev-tests/variables-entorno/ejemplo_env.py
import os

def main():
    # Acceder a las variables de entorno
    project_git_root = os.getenv("PROJECT_GIT_ROOT")
    project_local_root = os.getenv("PROJECT_LOCAL_ROOT")

    if project_git_root and project_local_root:
        print(f"PROJECT_GIT_ROOT: {project_git_root}")
        print(f"PROJECT_LOCAL_ROOT: {project_local_root}")

        # Concatenar PROJECT_LOCAL_CONFIG con "/configuracion_dispositivo.json"
        configuracion_dispositivo_file = os.path.join(project_local_root, "configuracion", "configuracion_dispositivo.json")

        # Imprimir el resultado
        print(f"configuracion_dispositivo_file: {configuracion_dispositivo_file}")
    else:
        print("Una o más variables de entorno no están definidas.")

if __name__ == "__main__":
    main()

File: scripts/dev-tests/variables-entorno/ejemplo_env.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
    // Acceder a las variables de entorno
    char* project_git_root = getenv("PROJECT_GIT_ROOT");
    char* project_local_root = getenv("PROJECT_LOCAL_ROOT");

    if (project_git_root != NULL && project_local_root != NULL) {
        printf("PROJECT_GIT_ROOT: %s\n", project_git_root);
        printf("PROJECT_LOCAL_ROOT: %s\n", project_local_root);

        // Concatenar PROJECT_LOCAL_CONFIG con "/configuracion_dispositivo.json"
        const char* file_suffix = "/configuracion/configuracion_dispositivo.json";

        // Declarar un buffer suficientemente grande para almacenar la cadena concatenada
        char configuracion_dispositivo_filename[100];

        // Inicializar la cadena
        strcpy(configuracion_dispositivo_filename, project_local_root);
        strcat(configuracion_dispositivo_filename, file_suffix);

        // Imprimir el resultado
        printf("configuracion_dispositivo_file: %s\n", configuracion_dispositivo_filename);
    } else {
        printf("Una o más variables de entorno no están definidas.\n");
    }

    return 0;
}
